nxc/protocols/ftp.py

```diff
@@ -80,16 +81,42 @@ def plaintext_login(self, username, password):
             host_id = self.db.get_hosts(self.host)[0].id
             self.db.add_loggedin_relation(cred_id, host_id)
 
-            if username in ["anonymous", ""] and password in ["", "-"]:
+            if username in ["anonymous", ""]:
                 self.logger.success(f"{username}:{process_secret(password)} {highlight('- Anonymous Login!')}")
             else:
                 self.logger.success(f"{username}:{process_secret(password)}")
-
+ 
```

Comment:
Please remove the space

Reply:
Removed space: https://github.com/RomanRII/NetExec/commit/f26b676c8ee94992006352737663352301ed201b